src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1288,7 +1307,7 @@ public void UpdateRigiditySliderState(int mutationPoints)
         if (editor == null)
             throw new InvalidOperationException("GUI not initialized");
 
-        if (mutationPoints >= Constants.MEMBRANE_RIGIDITY_COST_PER_STEP && editor.MovingOrganelle == null)
+        if (mutationPoints >= Constants.MEMBRANE_RIGIDITY_COST_PER_STEP && editor.MovingOrganelles == null)
```

Comment:
So multiple organelles can be moved at once? How does that work? The way I imagine it in my head is just much more confusing than only allowing to move a single organelle at once...

Reply:
The symmetry mode seems to not only get forgotten by the community but also by programmers while implementing moving/removing. I've modified the logic so that the symmetry mode works better. (e.g. moving/removing multiple organelles, placements in symmetry mode is now one action when undoing/redoing, better aborting when not having anough MP while placing with symmetry mode, etc..)

Reply:
It's not forgotten and for movement I think it's pretty useful to only move one organelle at once, that way you actually can do careful adjustments without having to toggle symmetry mode on and off while working.

Reply:
I'd say it makes more sense to allow symmetrical movement of organelles with symmetry mode. All hexes symmetric to the hovered mouse hex are also highlighted, therefore to me it makes more sense to apply all actions in symmetry, not only placement. Especially in freebuild mode moving six organelles at a time is much faster than six seperate movements. I guess a third opinion from @Revolutionary-Games/game-design wouldn't hurt.

Reply:
What about a case where symmetry is hovering over organelles with multiple hexes and some that are just a single one. If you move all at once you end up with a mishmash of different sized organelles to move, and it might be actually impossible to place them all at once.

Reply:
It works, you can try it. I have not found a bug with this.

Reply:
If you really want to ruin this functionality, could we please at least make this an option in the game options to select, and make it the non-default one, if players much prefer the move with symmetry we can make that the default?